input/fsh/SD_AdverseEvent.fsh

```diff
@@ -0,0 +1,44 @@
+Profile: CTCAdverseEvent
+Parent: AdverseEvent
+Title: "CTC Adverse Event"
+Id: ctc-adverse-event
+Description: "Profile of adverse event, using common terminology criteria (CTC)."
+* subject only Reference(Patient)
+* actuality = #actual
+* event 1..1
+* event from MedDRAVS (required)
+* event.text 1..1
+* severity 0..0  // replace with CTCAEGrade
+* extension contains CTCAEGrade named grade 0..1 
+* seriousness from AdverseEventSeriousnessVS (required)
+* suspectEntity.causality.assessment from AEA (required)
+
+Extension: CTCAEGrade
+Id: ctcae-grade
+Title: "CTC Adverse Event Grade"
+Description: "The grade associated with the severity of an adverse event, using CTCAE criteria. See https://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm"
+* value[x] only CodeableConcept
+* valueCodeableConcept from CTCAdverseEventGradeVS (required)
```

Comment:
a bit too tight. NCIt is used for CTCAE grade and is in context with the CTCAE term.